db/models/scores_justifications.py

```diff
@@ -0,0 +1,80 @@
+import uuid
+from datetime import datetime
+
+from db import db
+
+
+class ScoresJustifications(db.Model):
+    scores_justifications_id = db.Column(
+        "scores_justifications_id",
+        db.Text(),
+        default=lambda: uuid.uuid4(),
+        primary_key=True,
+    )
```

Comment:
Can we use `UUIDType` here instead? This will use the Postgres UUID native type
e.g.
```
id = sa.Column(
                UUIDType(binary=False),
                primary_key=True,
                default=uuid.uuid4
            )
```

Reply:
I agree with this